docs/resourse-cleanup.md

```diff
@@ -0,0 +1,64 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the `reclaimPolicy``
+as `Retain` will not delete the PV object , backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values , cephfs subvolume and rbd image.
+You need to cleanup metadata and image separately.
+
+## Steps
+
+
+### 1. Get volume ID from PV
+
+* '[$]kubectl get pvc pvc_name -n namespace -owide' to get pv_name <br>
+```
+$ kubectl get pvc mysql-pvc -owide -n prometheus
+NAME        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE   VOLUMEMODE
+mysql-pvc   Bound    pvc-bc537af8-67fc-4963-99c4-f40b3401686a   20Gi       RWO            csi-rbd        14d   Filesystem
+```
+
+### 2. Get omap keys
+
+* '[$]rados listomapkeys csi.volumes.default -p pool_name | grep pv_name' to get omapkey1 <br>
+```
+$ rados listomapkeys csi.volumes.default -p kube_csi | grep pvc-bc537af8-67fc-4963-99c4-f40b3401686a
+csi.volume.pvc-bc537af8-67fc-4963-99c4-f40b3401686a
+```
+* '[$]rados listomapkeys omapkey1 -p pool_name' to get omapkey2 <br>
+```
+$ rados listomapkeys csi.volume.dd2473d0-6a8c-11ea-9113-0ad59d995ce7 -p kube_csi
+csi.volname
+```
+
+
+### 3. Delete omap keys
+
+* '[$]rados rmomapkey omapkey1 omapkey2 -p pool_name' <br>
+```
+$ rados rmomapkey csi.volume.dd2473d0-6a8c-11ea-9113-0ad59d995ce7 csi.volname  -p kube_csi
```

Comment:
You should remove the entire OMap named csi.volume.<UUID>, instead of only the key within it. I would suggest replacing this with, `rados rm csi.volume.dd2473d0-6a8c-11ea-9113-0ad59d995ce7`

Reply:
done.